### ISSUE TITLE
Documentation says that static methods can be called as regular functions, but I don't see how from the documentation

### ISSUE THREAD

**Issue by khatchad** [python/cpython#117903]
# Documentation

The [documentation](https://docs.python.org/3/library/functions.html#staticmethod:~:text=Moreover%2C%20they%20can%20be%20called%20as%20regular%20functions%20(such%20as%20f()).) states that static methods can be called as "regular" functions, but I don't see how that's possible from the documentation. 

> Moreover, they can be called as regular functions (such as f()).

I tried the following example that didn't work:

```python
class C:
    @staticmethod
    def the_static_method(x):
        pass

the_static_method(5)
```

I get:

```
Traceback (most recent call last):
  File "....py", line 6, in <module>
    the_static_method(5)
NameError: name 'the_static_method' is not defined. Did you mean: 'staticmethod'?
```

### Linked PRs
* gh-117925
* gh-118509

**Comment by jamestwebber** [comment]
I think that is referring to the example below: if `f()` is defined as a regular function but also attached to `C` it could be called either way.

```
def the_static_method(x):
    pass

class C:
    the_static_method = staticmethod(the_static_method)

the_static_method(5)
C.the_static_method(5)  # calls the same function

**Comment by pochmann3** [comment]
Googling "Moreover, they can be called as regular functions" shows you're *not* the first to wonder about that, and finds for example [this](https://discuss.python.org/t/staticmethod-called-outside-class-scope/15007/5).

**Comment by terryjreedy** [comment]
In general, questions about using Python should be directed to help forums such as https://discuss.python.org/c/users/7 (or stackoverflow, where there have been numerous questions about `staticmethod`).  However the quoted sentence from the [staticmethod entry](https://docs.python.org/3/library/functions.html#staticmethod), especially "(such as f())"), is actively misleading.  The latter is only valid in class code.  I am thinking about a suggested improvement.

**Comment by serhiy-storchaka** [comment]
It means that the staticmethod descriptors are callable:
```py
class C:
    @staticmethod
    def the_static_method(x):
        pass
    the_static_method(5)
```
It should be worded more directly ("the staticmethod descriptors are callable"), and you usually can ignore this feature, it is almost never used.

**Comment by khatchad** [comment]
> In general, questions about using Python should be directed to help forums ... 

I've reworded the issue.

**Comment by serhiy-storchaka** [comment]
Fixed by #117925.
